lib/demo-block-templates/single.html

```diff
@@ -0,0 +1,2 @@
+<!-- wp:post-title /-->
+<!-- wp:post-content /-->
```

Comment:
Seems like this is an unrelated change, do we want it or should we remove it from the PR?

Reply:
Oops, deleted.